pm/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+list(APPEND src_pm
+  process_pm.f90)
+
+add_executable(process_pm.exe ${src_pm})
+target_link_libraries(process_pm.exe PRIVATE ${PROJECT_NAME}::ncio
+                                          PRIVATE ${PROJECT_NAME}::esggrid_util
+                                          PRIVATE ${PROJECT_NAME}::pesglib
+                                          PRIVATE NetCDF::NetCDF_Fortran
+                                          PRIVATE gsi::gsi
```

Comment:
May not need ncio,esggird_util,pesglib, gsi

Reply:
You can delete them and try again.

Reply:
done